### ISSUE TITLE
Offsets to operating expenditures pulling the wrong data on committee page

### ISSUE THREAD

**Issue by jwchumley** [18F/openFEC#2249]
https://beta.fec.gov/data/committee/C00495861/?cycle=2016&tab=summary

It looks like the offsets value is actually displaying the operating expenditures value on this page (for all committees)
In this example:
The offsets amount: $26,672,032.23 should be $3,421,567

The $26 million figure is actually the Operating Expenditures amount for this committee. It is correctly placed in the disbursements block.

@hcaofec @PaulClark2

**Comment by noahmanger** [comment]
Oh yep that looks like a bug. This looks like the SQL might be wrong in the API. 

For pac-party totals ([source](https://github.com/18F/openFEC/blob/develop/data/sql_updates/reports_pac_party.sql#L48)):
```
    ttl_op_exp_per as offsets_to_operating_expenditures_period,
    ttl_op_exp_ytd as offsets_to_operating_expenditures_ytd,
```

Compared to house-senate totals:
```
    offsets_to_op_exp_per as offsets_to_operating_expenditures_period,
    offsets_to_op_exp_ytd as offsets_to_operating_expenditures_ytd,
```

**Comment by LindsayYoung** [comment]
Going from the last table to this view, we need to do the greatest function on items that appear multiple times in the form.

**Comment by LindsayYoung** [comment]
closed by # #2281
